### ISSUE TITLE
Rename noir_js-types to be noir_js-bckend-types

### ISSUE THREAD

**Issue by kevaundray** [noir-lang/noir#2895]
### Problem

This is a tracking issue for:

- https://github.com/noir-lang/noir/pull/2893#discussion_r1341285326
- https://github.com/noir-lang/noir/pull/2893#discussion_r1341285611

### Happy Case

Solve those issues

### Would you like to submit a PR for this Issue?

No

**Comment by Savio-Sou** [comment]
Would this be a "good first issue"?

**Comment by kobyhallx** [comment]
why is this a high priority? Why we should rename this package?

**Comment by TomAFrench** [comment]
> Would this be a "good first issue"?

Not really. The work on issues like this is deciding what the names should be rather and why it should change than doing a search-and-replace.

**Comment by Savio-Sou** [comment]
> deciding what the names
Ah we should switch this into a GitHub Discussion then (now that we have it).
